openapi/src/parts/workspace.yaml

```diff
@@ -545,6 +545,9 @@ components:
         azureContext:
           description: Azure context, if one exists
           $ref: '#/components/schemas/AzureContext'
```

Comment:
We do not want/need this for AWS.  The AWS context creation code path gets its data from configuration.  We may want to specify a landing zone ID or description later, but we are not there yet.

Reply:
AWS account number - Added it as a placeholder, but landing zoneId makes more sense here.

Reply:
`s/Azure/AWS` in description